### DISCUSSION TITLE
Mastodon for iOS is in Finnish for Swedish people

### DISCUSSION THREAD

**Discussion by mikaelf** [mastodon/mastodon#18101] [Q&A]
I use my iPhone with the system settings language set to `Svenska` (`Swedish`). The labels in the Mastodon for iOS app are shown in Finnish, a language most Swedes doesn't know. In lack of a Swedish translation, defaulting to English would be better.

I believe the issue is that the Finnish translation in the Mastodon for iOS app is in `Svenska (Finland)` (`Swedish (Finland)`).

## Workaround
There's a workaround by going to the system settings for Mastodon for iOS (n.b. not the settings inside the app, but in the system settings) and changing the preferred app language setting for Mastodon from `Swedish (Finland)` to `English`.

## Context
In Sweden, Finnish is a minority language, known by roughly 2% of the population. In Finland, Swedish is an official language that at least historically was widely known among the population.

## Issue
Issue in the mastodon-ios repo: https://github.com/mastodon/mastodon-ios/issues/384

**Comment by kahari** [comment]
Workaround doesn’t work for me. There is no language settings in the system settings for Mastodon app. I’m stuck with Finnish 😂

iOS version 15.4.1
Mastodon version 1.3.1

**Reply by mikaelf** [reply]
Weird! Seems like there's progress in [the issue](https://github.com/mastodon/mastodon-ios/issues/384) so soon you won't have to. You could set your phone language to English in the meanwhile.
